02-library/cs506/det.py

```diff
@@ -0,0 +1,13 @@
+def determinant(A):
+  """takes a 2D matrix as a list of atleast size 2x2"""
+  if len(A) == 2 and len(A[0]) == 2:
+    return A[0][0]*A[1][1] - A[0][1]*A[1][0] 
+  else:
+    terms = []
```

Comment:
this is a bit of a waste of space... why not do 
```suggestion
    det = 0
```